### PR TITLE
.github: Add label checklist to templates

### PR DESCRIPTION
**Pull request by fyrchik**
Signed-off-by: Evgenii Stratonikov <evgeniy@morphbits.ru>

### CODE REVIEW

**Review by carpawell — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -18,3 +18,11 @@ assignees: ''
 
 ## Additional context
 <!-- Add any other context or screenshots about the feature request here. -->
+
+## Don't forget to add labels!
+- component label (`neofs-adm`, `neofs-storage`, ...)
+- issue type (`enhancement`, `refactor`, ...)
+- `goodfirstissue`, `helpwanted` if needed
+- does this issue belong to an epic?
+- priority (`P0`-`P4`)
```
Comment (carpawell) [inline .github/ISSUE_TEMPLATE/feature_request.md:27]: "if not triaged yet" or smth like that?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -18,3 +18,11 @@ assignees: ''
 
 ## Additional context
 <!-- Add any other context or screenshots about the feature request here. -->
+
+## Don't forget to add labels!
+- component label (`neofs-adm`, `neofs-storage`, ...)
+- issue type (`enhancement`, `refactor`, ...)
+- `goodfirstissue`, `helpwanted` if needed
```
Comment (carpawell) [inline .github/ISSUE_TEMPLATE/feature_request.md:25]: about the triage?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -18,3 +18,11 @@ assignees: ''
 
 ## Additional context
 <!-- Add any other context or screenshots about the feature request here. -->
+
+## Don't forget to add labels!
+- component label (`neofs-adm`, `neofs-storage`, ...)
+- issue type (`enhancement`, `refactor`, ...)
```
Comment (realloc) [inline .github/ISSUE_TEMPLATE/feature_request.md:24]: `bug` is the most important label here =)

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 2 above)
Comment (fyrchik) [reply .github/ISSUE_TEMPLATE/feature_request.md:25]: `triage` is added by default to all newly created issuses

Comment (carpawell) [reply .github/ISSUE_TEMPLATE/feature_request.md:25]: i meant removing it if it is already discussed. but yes, forget about the default behavior

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 1 above)
Comment (fyrchik) [reply .github/ISSUE_TEMPLATE/feature_request.md]: Fixed

**Review by carpawell — APPROVED** [review]

**Review by cthulhu-rider — APPROVED** [review]

**Pull request merged**